File: w10/d02/Christopher_Bajorin/morning_exercise.js
// "Peter Parker" regex
// flip and print to node

// // vanilla name
// var name = "Peter Parker"
// var reversedname = name.replace(/(\w+)\s(\w+)/, '$2, $1')
// console.log(reversedname)

// complicated name
// var name = "Eva-Lotte O'Malley"
var name = "Declan Van Welie"
var reversedname = name.replace(/([\w\W]+)\s([\w\W\s]+)/, '$2, $1')
// var reversedname = name.replace(/([^ ]+)\s([\w\W]+)/, '$2, $1')
console.log(reversedname)

File: w10/d02/Christopher_Lee/morning_exercise.rb
# var objRegExp = /(\w+)\s(\w+)/;
# var fullName = "Jane Doe";
# var reverseName = fullName.replace(objRegExp, "$2, $1");

"Peter Parker".replace(/(\w+)\s(\w+)/, "$2, $1");


"Eva-Lotte O'Malley".replace(/([a-zA-Z'-]+)\s([a-zA-Z'-]+)/, "$2, $1");

"Eva-Lotte O'Malley".(/([\w-']+)\s()([\w-']+)/, "$2, $1");
